samples/boards/stm32/power_mgmt/blinky/sample.yaml

```diff
@@ -10,5 +10,6 @@ tests:
         - "Device ready"
     filter: dt_compat_enabled("zephyr,power-state") and
             dt_enabled_alias_with_parent_compat("led0", "gpio-leds") and
-            dt_compat_enabled("st,stm32-lptim")
+            dt_compat_enabled("st,stm32-lptim") and
+            dt_chosen("zephyr,pm-timer")
```

Comment:
References to `zephyr,pm-timer` to be removed as it's not used anymore

Reply:
removing the commits  related to the the multi-instance of lptim and the method for choosing one.